core.coffee

```diff
@@ -230,7 +230,7 @@
 			apiPrefix: '/'
 			passthrough: {}
 			passthroughByMethod: {}
-			constructor: (params) ->
+			constructor: (params, backendParams) ->
```

Comment:
So we now allow instantiation of the backend class directly? Previously wasn't it the case that the 'backend' subclassed something and the user would construct just the core?

I think this requires at least a comment saying that this sig is for information only, as right now it just looks like an unused var - unfort we can't have an abstract class :(